xbmc/music/MusicDatabase.cpp

```diff
@@ -7486,9 +7491,9 @@ void CMusicDatabase::UpdateTables(int version)
   // The original db version when the tags were scanned, and the minimal db version needed are
   // later used to determine if a forced rescan should be prompted
 
-  // The last schema change needing forced rescanning was 60.
-  // Mostly because of the new tags processed by v17 rather than a schema change.
-  SetMusicNeedsTagScan(60);
+  // The last schema change needing forced rescanning was 73.
+  // This is because Kodi can now read and process extra tags involved in the creation of box sets.
+  SetMusicNeedsTagScan(73);
```

Comment:
OK force a rescan, otherwise no one will see anything.
Although one idea was during development to allow the rescanning to be optional when possible (even though to test the new features they need to rescan) and then bump this for say alpha and apply all the tag additions in one go.
There are going to be more tag additions (I have a list ready to do), so may decide to remove this for the Kodi PR depending on how near to PR I get with those additions.